src/ontology/cl-edit.owl

```diff
@@ -29451,6 +29452,19 @@ SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000005444))
 SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015665))
 
+# Class: obo:CL_4023122 (oxytocin receptor sst GABAergic cortical interneuron (Mus musculus))
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023122 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25303526"^^xsd:string) <http://www.ebi.ac.uk/cellline/definition> obo:CL_4023122 "An interneuron located in the mouse cerebral cortex that express the oxytocin receptor. These interneurons also express somatostatin.")
```

Comment:
Hi @shawntanzk , is the ebi celline definition thing ok, or was it a setting left from previous edits maybe for BDS?

Reply:
oh that is 100% wrong - I know what this is, will make a change, thanks for spotting